source/features/list-prs-for-file.tsx

```diff
@@ -34,7 +34,7 @@ function getDropdown(prs: number[]): HTMLElement {
 						data-url={buildRepoURL('issues', prNumber)}
 						data-id={`rgh-pr-${prNumber}`}
 					>
-						<a className="dropdown-item" href={getPRUrl(prNumber)}>
+						<a className="dropdown-item" href={getPRUrl(prNumber)} data-pjax="#js-repo-pjax-container">
```

Comment:
If `#repo-content-pjax-container` is used GitHub will 304 redirect to return `#js-repo-pjax-container"` which breaks the page, so just use the latter directly